lib/registration.pm

```diff
@@ -75,8 +77,26 @@ our %SLE15_DEFAULT_MODULES = (
     sles4sap => 'base,desktop,serverapp,ha,sapapp',
 );
 
+our %ADDONS_REGCODE = (
```

Comment:
I think we'd better not to set extensions not owned by migration, if add them then please support verify log for HA,GEO,RT, etc.

Reply:
I will keep it. When I have time, I will verify run their cases.